kornia/color/rgb.py

```diff
@@ -2,6 +2,52 @@
 import torch.nn as nn
 
 
+class RgbToRgba(nn.Module):
+    r"""Convert image from RGB to RGBA.
+
+    Add an alpha channel to existing image of special value named alpha_val
+
+    args:
+    image (torch.Tensor): A Three Channeled Image Tensor
```

Comment:
guess you need indentation in parameters, otherwise docs won't render correctly in the docs.